odata-protocol/10 Context URL.md

```diff
@@ -242,14 +253,15 @@ http://host/service/$metadata#Customers(Address,Orders,Model.VipCustomer/Preferr
 Context URL templates:
 
     {context-url}#{entity-set}{/type-name}{select-list}/$entity
-    {context-url}#{singleton}{select-list}
+    {context-url}#{single-entity}{select-list}
     {context-url}#{type-name}{select-list}
 
-If a single entity contains a subset of properties, the parenthesized
+If an entity contains a subset of properties, the parenthesized
 comma-separated list of the selected defined or dynamic properties,
 instance annotations, navigation properties, functions, and actions is
 appended to the `{entity-set}` after an optional type-cast segment and
-prior to appending `/$entity`. If the response is not a subset of a
+prior to appending `/$entity`, or to the `{single-entity}`.
+If the response is not a subset of a
 single entity set, the `{select-list}` is instead appended to the
 `{type-name}` of the returned entity.
```

Comment:
This cannot cannot occur, because the response in this section is always _one_ entity, not a (sub)set.